class-instant-articles-signer.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Facebook Instant Articles for WP.
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @package default
+ */
+
+/**
+ * Class responsible for handling the signature of graph rescrape calls
+ *
+ * @since 4.1
+ */
+class Instant_Articles_Signer {
+
+  const PUBLIC_KEY_OPTION = 'instant-articles-rescrape-public-key';
+  const PRIVATE_KEY_OPTION = 'instant-articles-rescrape-private-key';
+  const PUBLIC_KEY_PATH = '.well-known/graph-api/apikey.pub';
```

Comment:
Is this a fixed place? Should we make it by default this path, but allow configuration in the settings page.

Reply:
This is a fixed place, yes. Will not be configurable.